app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+import classNames from 'classnames';
+
+let Comment = React.createClass({
+  propTypes: {
+    comment: React.PropTypes.shape,
```

Comment:
We can still make this more specific!

```js
comment: React.PropTypes.shape({
  replies: React.PropTypes.arrayOf(React.PropTypes.string),
  // etc.
})
```

Make use of `isRequired` as well! Here's the documentation for PropTypes: https://facebook.github.io/react/docs/reusable-components.html

Also, consider adding comments explaining each property like this: https://github.com/pianoshelf/webclient-react/blob/master/app/components/Search/utils/FilterGroup.jsx#L17

It may seem tedious but it goes a long way to helping future developers understand how each component works without going through the code. It's worth noting that none of this matters in production, but in our dev environments, React prints a warning message whenever we violate a PropType rule.